tests/test_codecs.py

```diff
@@ -390,7 +390,6 @@ def test_decoder_state(self):
                                          u, u.encode(self.encoding))
 
     def test_lone_surrogates(self):
-        self.assertRaises(UnicodeEncodeError, "\ud800".encode, self.encoding)
```

Comment:
Hello @jonkiparsky !
Can you tell me why is this test being removed?
It looks like this was passing before the changes in this PR.